packages/gatekeeper/README.md

```diff
@@ -37,6 +37,35 @@ class App extends Component {
 export default App
 ```
 
+You can use the logout component to log out of the authentication server as well. This can be done by passing a logoutURL prop to ConnectedLogout as follows:
+
+```tsx
+import { ConnectedLogout } from `@onaio/gatekeeper`;
+
+class App extends Component {
+  render() {
+    return (
+     ...
+        <Router>
+          <div className={'main-container'}>
+            <Switch>
+              <Route path="/login" component={Login} />
+              <Route path="/logout" component={() => (
+              <ConnectedLogout
+                {...({ logoutURL: 'https://www.auth2serversURL/logout' } as Partial<LogoutProps>)}
```

Comment:
Instead of `Partial<LogoutProps>` can we please just put in all the expected props?  One good side effect of that is to improve the docs for this package.